release_notes/Service_Fabric_ReleaseNotes_90.md

```diff
@@ -0,0 +1,153 @@
+| 22. | **Bug Fix** | IActorService.GetActors fails in KVS due to case sensitivity | **Brief Description**: GetActors API returns a single page infinitely if a KVS actor service has upper case actors as well as lower case actors.<br>**Impact**: GetActors enumeration API<br>**Fix**: GetActors API will enumerate all actors in the KVS service with upper and lower case actors |
+
+<br>
+
+## Future Retirement and Deprecation Path Callouts
+
+| Support | Option | Details |
+|-----------------|--------|---------|
+| **Retired support for ASP.NET Core 1.0** | Migrate usage to ASP.NET Core 2.1 or greater | Service Fabric ASP.NET Core packages are built against ASP.NET Core 1.0 binaries which are out of support. Starting Service Fabric 8.2, we will be building Service Fabric ASP.NET Core packages against .NET Core 3.1, .NET Core 5.0, .NET Core 6.0, .NET Framework 4.6.1. As a result, they can be used only to build services targeting .NET Core 3.1, .NET Core 5.0, .NET Core 6.0, >=.NET Framework 4.6.1 respectively.<br>For .NET Core 3.1, .NET Core 5.0, .NET Core 6.0, Service Fabric ASP.NET Core will be taking dependency on Microsoft.AspNetCore.App shared framework, whereas for NetFx target frameworks >=.NET Framework 4.6.1, Service Fabric ASP.NET Core will be taking dependency on ASP.NET Core 2.1 packages.<br>The package Microsoft.ServiceFabric.AspNetCore.WebListener will no longer be shipped, Microsoft.ServiceFabric.AspNetCore.HttpSys package should be used instead.|
+| **Deprecated support for .NET Core 2.x** | Migrate usage to .NET Core 3.1 | .NET Core runtime LTS 2.1 runtime is out of support as of Aug 21, 2021, and .NET Core runtime 2.2 is out of support from Dec 2019.<br>Service Fabric releases after those dates will drop support for Service Fabric applications running with .NET Core 2.1 & .NET Core 2.2 runtimes respectively.<br>Current applications running on .NET Core 2.x runtime will continue to work, but requests for investigations or request for changes will no longer receive support. Service Fabric .NET SDK will take a dependency on .NET runtime 3.x features to support Service Fabric .NET Core applications. |
```

Comment:
```suggestion
| **Deprecated support for .NET Core 2.x** | Migrate usage to .NET Core 3.1 | .NET Core runtime LTS 2.1 runtime is out of support as of Aug 21, 2021, and .NET Core runtime 2.2 is out of support since Dec 2019.<br>Service Fabric releases after those dates will drop support for Service Fabric applications running with .NET Core 2.1 & .NET Core 2.2 runtimes respectively.<br>Current applications running on .NET Core 2.x runtime will continue to work, but requests for investigations or request for changes will no longer receive support. Service Fabric .NET SDK will take a dependency on .NET runtime 3.x features to support Service Fabric .NET Core applications. |
```

Reply:
^"from"-->"since"